spec/controllers/api/v2/migrations/splits_controller_spec.rb

```diff
@@ -19,12 +19,13 @@
 
     describe '#create' do
       it "creates a new split with desired weightings" do
-        post :create, params: { name: 'default_app.foobar', weighting_registry: { foo: 10, bar: 90 } }
+        post :create, params: { name: 'default_app.foobar', owner: 'b4c', weighting_registry: { foo: 10, bar: 90 } }
 
         expect(response).to have_http_status(:no_content)
         split = Split.find_by(name: 'default_app.foobar', owner_app: default_app)
         expect(split).to be_truthy
         expect(split.registry).to eq 'foo' => 10, 'bar' => 90
+        expect(split.owner).to eq 'b4c'
```

Comment:
Nitpicking, but no need to use internal team names on a public repo 😉 

```suggestion
        post :create, params: { name: 'default_app.foobar', owner: 'team-name', weighting_registry: { foo: 10, bar: 90 } }

        expect(response).to have_http_status(:no_content)
        split = Split.find_by(name: 'default_app.foobar', owner_app: default_app)
        expect(split).to be_truthy
        expect(split.registry).to eq 'foo' => 10, 'bar' => 90
        expect(split.owner).to eq 'team-name'
```